src/main/kotlin/org/malachite/estella/people/api/UserController.kt

```diff
@@ -38,7 +39,7 @@ class UserController(
             return ResponseEntity(Message("Invalid password"), HttpStatus.BAD_REQUEST)
 
         val token = securityService.getTokens(user, response)
-        return token?.let { ResponseEntity(SuccessMessage, HttpStatus.OK) }
+        return token?.let { ResponseEntity(Token(token), HttpStatus.OK) }
```

Comment:
Tu chyba mogło by być ```Token(it)```

Reply:
Szczerze, jak ma być to jedyna rzecz, to nie chce mi się już zmieniać XDD